Sources/Core/SourceFile.swift

```diff
@@ -344,17 +352,26 @@ extension SourceFile {
     }
 
     /// The owner of all instances of `Storage`.
-    private static var allInstances = SharedMutable<[URL: Storage]>([:])
+    private static let allInstances = SharedMutable<[URL: Storage]>([:])
 
     /// Creates an alias to the instance with the given `url` if it exists, or creates a new
     /// instance having the given `url` and the text resulting from `makeText()`.
+    /// Storage instances are cached based on url, where `replaceExisting` defines if an
+    /// existing entry should be replaced with new content, otherwise the storage get the
+    /// content from the previously cached entry.
```

Comment:
```suggestion
    ///
    /// Storage instances are cached based on url, where `replaceExisting` defines if an
    /// existing entry should be replaced with new content, otherwise the storage get the
    /// content from the previously cached entry.
```
We have to add one "empty"  line after the first sentence to let Swift Docs understand we're starting a detailed discussion. The first paragraph is what ends up in the title of Xcode's quick help (and I guess other IDEs).